app/src/UI/Map.tsx

```diff
@@ -174,18 +174,17 @@ const MapMarkers = (props: any) => {
     }
   };
 
-
-  const Meese = (props: any) => (
-    <>
-      {mooseArray.map((moose: any, index: number) => {
-        const position = getOffsetLocation(index, markerPosition);
-        const mooseIcon = getMooseIcon(moose, "Current");
-        return (
-          <Marker key={index} position={position} icon={mooseIcon} />
-        );
-      })}
-    </>
-  );
+  // const Meese = (props: any) => (
+  //   <>
```

Comment:
delete too?

Reply:
Leaving it in for now, as we will need to display moose icons later (albeit not at an exact lat/lon position)